src/components/frame/useFrame.ts

```diff
@@ -87,9 +91,13 @@ export const useChildFrame = function (
         })
         .catch((err) => {
           trace("connectToChild failed: ", err);
-          setStatus("DISCONNECTED");
+          if (err.errors.map((e: Error) => e.message).some((m: string) => m.includes("timed out"))) {
```

Comment:
how come this error is different from the other ? Is there any doc on this in the lib ?

Reply:
It's different because `Promise.any()` will return an [AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError).

Reply:
Oh yes, they do have a `code` property in their Error objects. Will utilise that.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError

Reply:
ah nice, let's use this indeed :)